packages/devextreme/js/__internal/ui/chat/chat.ts

```diff
@@ -183,14 +207,26 @@ class Chat extends Widget<Properties> {
   }
 
   renderMessage(message: Message = {}): void {
-    const { items } = this.option();
+    // @ts-expect-error
+    const dataSource = this.getDataSource();
 
-    const newItems = [...items ?? [], message];
+    if (!isDefined(dataSource)) {
+      const { items } = this.option();
 
-    this.option('items', newItems);
+      const newItems = [...items ?? [], message];
+      this.option('items', newItems);
+      return;
+    }
+
+    dataSource.store().insert(message).done(() => {
+      dataSource.load();
```

Comment:
Is it required? Seems like it means that we'll send addition request after single message rendering, so it would be great if we can omit it

Reply:
We discussed it separately. The details can be found in the corresponding team's post.